client/my-sites/plugins/use-plugins/index.ts

```diff
@@ -107,42 +89,32 @@ const usePlugins = ( {
 			enabled: category === 'featured' && wpcomEnabled,
 		} ) as WPCOMResponse;
 
-	const featuredPlugins = useMemo(
-		() => ( featuredPluginsRaw as Plugin[] ).map( updateWpComRating ),
-		[ featuredPluginsRaw ]
-	);
```

Comment:
I would rename the `featuredPluginsRaw` as `featuredPlugins`, this way you wouldn't need to change the next occurrences. 

FWIR, I named `featuredPluginsRaw` just because I would recalculate the rating, but it was called `featuredPlugins` before.

Reply:
As you suggested, I've applied a similar change on #72383, but removing the `raw` part as I mentioned.

Reply:
> would rename the `featuredPluginsRaw` as `featuredPlugins`, this way you wouldn't need to change the next occurrences.
> 
> FWIR, I named `featuredPluginsRaw` just because I would recalculate the rating, but it was called `featuredPlugins` before.

Thanks, you're right it sounds more straightforward that way! I'll update this PR too soon, this way the branches hopefully won't conflict when merging. 😃

Reply:
> Why don't we have the number of ratings in all views? It's somewhat awkward to show it on search but not on the paid plugins list.

I don't know the original cause for this, I went back to check the Figma designs and those have the number of installs on the cards. I think it's meant to be only missing when we don't have the number:
f84fqtCG4pCnCh2Hq0But5-fi-5669%3A41901
<img width="1014" alt="CleanShot 2023-01-24 at 14 37 47@2x" src="https://user-images.githubusercontent.com/11555574/214323338-cc0f3ded-9989-4614-bb36-cfab01b6e7c6.png">

I'll update the PR to show it in all views.

Reply:
Actually, the Calypso side looks good, rating numbers aren't hidden on purpose:
https://github.com/Automattic/wp-calypso/blob/7b309f3b6110da1abb41cce4894c5f984443af13/client/my-sites/plugins/plugin-ratings/index.jsx#L119-L121

I think this is caused by some views still being served from the static catalog, which doesn't have the `num_ratings` field and it doesn't store the number of ratings in another field either. For search, we have the data now in the ES index and that's shown correctly. So this is a non-issue, it will be solved by migrating to the dynamic catalog.

Reply:
Great, thanks for diving into this.